src/content/docs/logs/logs-everywhere/ruby-telemetry-context.mdx

```diff
@@ -0,0 +1,41 @@
+* Navigate within multiple types of telemetry data, and have the data correlate back to the original issue.
+* Easily drill down into more detailed information from the same place in the UI.
+* Find the log lines that you need to identify and resolve a problem.
+
+For example, with telemetry in context, you can easily see [log patterns](/docs/logs/ui-data/find-unusual-logs-log-patterns), or show [surrounding logs](/docs/logs/troubleshooting/find-issues-cause-or-impact-surrounding-logs) at the time an error occurred in your app.
+
+## Get started [#enable]
+
+To enable telemetry in context for APM apps monitored by Ruby, install or update to [Ruby agent version 8.6.0  or higher](https://docs.newrelic.com/docs/release-notes/agent-release-notes/ruby-release-notes). **REVIEWER: version tbd**
+
+That's it! To make the most of your logging data:
+
+* Explore the logging data across your platform with our [Logs UI](/docs/logs/log-management/ui-data/use-logs-ui/).
+* See your logs in context of your app's performance in the [APM UI](/docs/logs/log-management/ui-data/use-logs-ui/#links). Troubleshoot [errors](/docs/apm/apm-ui-pages/error-analytics/errors-page-find-fix-verify-problems/) with [distributed tracing](/docs/apm/distributed-tracing/ui-data/understand-use-distributed-tracing-data), stack traces, application logs, and more.
+* Get deeper visibility into both your application and your platform performance data by forwarding your logs with our [infrastructure monitoring agent](/docs/logs/enable-log-management-new-relic/enable-log-monitoring-new-relic/forward-your-logs-using-infrastructure-agent/). Review your [infrastructure logs](/docs/logs/log-management/ui-data/use-logs-ui/#links) in the UI.
+* Set up [alerts](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-alert-conditions/).
+* [Query your data](/docs/query-your-data/explore-query-data/get-started/introduction-querying-new-relic-data/) and [create dashboards](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/).
+
+**REVIEWER: These bullets come from the original Ruby logs in context doc. Will need to be updated.**
+
+## Optional configuration [#config]
+
+When you update to telemetry in context, three changes are made to your configuration file:
+
+* Application log forwarding enabled by default
+* Maximum log sample stored (10,000 lines by default)
+* Logging metrics, including `trace.id`, `span.id`, `entity_guid`, and `hostname`
+
+To collect other metadata, or to change these default settings, update your [`newrelic.yml` configuration file](https://docs.newrelic.com/docs/apm/agents/ruby-agent/configuration/ruby-agent-configuration/).
+
+**REVIEWER: Does anything else in the [original Ruby logs in context doc](https://docs.newrelic.com/docs/logs/logs-context/configure-logs-context-ruby) need to be added to this?**
```

Comment:
I'd like to take a closer look at the [incompatible gems](https://docs.newrelic.com/docs/logs/logs-context/configure-logs-context-ruby/#incompabilities) section to see if they are also incompatible with this feature. Some of them I haven't heard about/tested before. There's another one I recently learned about called Log4r, which is like Log4j but for Ruby.  

As an aside, do you know if there's an intention to remove support for the DecoratingLogger? I wonder if anyone might still need access to that information if they're using that procedure? (Perhaps @jasonrclark would know?)

Reply:
I have not heard anything specific about the other ones listed in https://docs.newrelic.com/docs/logs/logs-context/configure-logs-context-ruby . Do you know, Jason? Originally I was thinking that we could say something along the lines of, "If necessary, you can use the original logs in context options [link to this doc] but be aware that you will not have as much telemetry data with those options." or something like that.